packages/eslint-config-jwp/.depcheckrc.yaml

```diff
@@ -0,0 +1,8 @@
+ignores: [
```

Comment:
Do you think we could store linting configs somewhere in packages/configs?

Reply:
Can you elaborate?

Reply:
I mean having one folder `configs` in the packages with `eslint`, `postcss` and other linting configs inside.

```
/packages/configs
    /eslint-config-jwp
    /postcss-config-jwp
    /...
```

Or maybe on the top level:

```
/configs
    /eslint-config-jwp
    /postcss-config-jwp
    /...
```

Reply:
The second suggestion is easier because of the workspaces configuration in the package.json.